plugins/wallabag.koplugin/main.lua

```diff
@@ -664,25 +664,33 @@ function Wallabag:downloadArticle(article)
     local file_ext = ".epub"
     local item_url = "/api/entries/" .. article.id .. "/export.epub"
 
-    -- The mimetype is actually an HTTP Content-Type, so it can include a semicolon with stuff after it.
-    -- Just in case we also trim it, though that shouldn't be necessary.
-    -- A function represents `null` in our JSON.decode, because `nil` would just disappear.
-    -- We can simplify that to not a string.
-    local mimetype = type(article.mimetype) == string and util.trim(article.mimetype:match("^[^;]*")) or nil
+    -- The mimetype is actually an HTTP Content-Type, so we remove everything from the `;` onward
```

Comment:
The original comment is better. It explains the why, while this just repeats the code below.